### ISSUE TITLE
Water Source blocks as well as Clay blocks being generated on its own?

### ISSUE THREAD

**Issue by Aquintus** [NillerMedDild/Enigmatica2Expert#1655]
**Modpack version**
1.77

**Issue**
We made a "Slime farm" for mana generation via Dark Utilities Monoliths of Spawning and Random Things Slime Cubes and some Cyclic Fans at around 200 height in a plains biome with basic cobble stone as the building block. But for some reason the area where slimes hit the ground to be shoved towards the Botania Flowers gets clogged with water source blocks as well as clay.
 
![image](https://user-images.githubusercontent.com/40473897/79571976-71d1c000-80bc-11ea-8979-4980cfdb45ae.png)
The layers where the slimes are being spawned strangely enough do not have that issue, so it's only where the slimes will land. 

Kind Regards

**Comment by NillerMedDild** [comment]
I have nooo idea what causes that ^^'

**Comment by Aquintus** [comment]
I would assume that it is some config setting or just some weird mod mechanic in general. I don't really expect anyone to go through all of those in order to be able to pin point it down. Just felt like reporting it anyone so anyone that end up having the same issue can google it and end up in this issue report :D

**Comment by bibo38** [comment]
This is just weird :exploding_head: . @Aquintus Could you try to make this reproducible (a map would be the best way with maybe some info how long it takes the clay to appear on your machine)? I would be interested to spend some time to find some weird mod mechanics or bugs.

**Comment by Aquintus** [comment]
I don't have the world as it was on a server hosted by someone else but all the blocks that we used + biome + general height and modpack version are in my post so u could just make a similar setup :D

And for how long it takes for the clay and what to appear is kinda hard to say. we never ended up using the setup and just let it sit there. it might have been running for 2 RL days non-stop when i first realized it. Maybe more, maybe less. And after we cleaned it all up it went happened once more, so it was something we were able to reproduce. 

Not sure what else there is i could tell u. I placed some glass above it if it was some weird downfall thing and i also replaced the cobblestone with hay where the slimes would fall down but shortly after that i was left alone to play and i stopped due to that reason ^^
